cocos/ui/edit-box/EditBox-mac.mm

```diff
@@ -277,10 +272,6 @@ void init(const cc::EditBox::ShowInfo &showInfo) {
     }
 
     EditBox::_isShown = false;
-    if (EditBox::mouseDownListenerId != 0) {
-        EventDispatcher::removeCustomEventListener(EVENT_MOUSE_DOWN, EditBox::mouseDownListenerId);
-        EditBox::mouseDownListenerId = 0;
-    }
```

Comment:
So, can remove corresponding header files of EventDispatcher and related classes if have.